app/stories/pix-pagination.stories.js

```diff
@@ -119,4 +132,14 @@ export const argTypes = {
       defaultValue: { summary: 'fr' },
     },
   },
+  isCondensed: {
+    name: 'isCondensed',
+    description: 'Permet de mettre le composant en mode `condensé` pour le mode desktop',
```

Comment:
La description n'apporte que peu d'info supplémentaire, je propose

```suggestion
    description: 'En desktop, retire le contrôle du nombre d'élements par page pour simplifier l'usage',
```